doc/release/release_0.20.rst

```diff
@@ -0,0 +1,539 @@
+- Remove deprecated parameter value ``method='_lorensen'`` in ``skimage.measure.marching_cubes``
+  (`#6230 <https://github.com/scikit-image/scikit-image/pull/6230>`_).
+
+Bug fixes
+---------
+.. Only fixes to unexpected behavior in our public API. Exclude things such as fixes to the CI or documentation.
+
+- Fix round-off error in ``skimage.exposure.adjust_gamma``
+  (`#6285 <https://github.com/scikit-image/scikit-image/pull/6285>`_).
+- Round and convert output coordinates of ``skimage.draw.rectangle`` to ``int`` even if the input coordinates use ``float``.
+  This fix ensures that the output can be used for indexing similar to other draw functions
+  (`#6501 <https://github.com/scikit-image/scikit-image/pull/6501>`_).
+- Avoid unexpected exclusion of peaks near the image border in ``skimage.feature.peak_local_max`` if the peak value is smaller 0
+  (`#6502 <https://github.com/scikit-image/scikit-image/pull/6502>`_).
+- Avoid anti-aliasing in ``skimage.transform.resize`` by default when using nearest neighbor interpolation (``order == 0``) with an integer input data type
+  (`#6503 <https://github.com/scikit-image/scikit-image/pull/6503>`_).
+- Use mask during rescaling in ``skimage.segmentation.slic``.
+  Previously, the mask was ignored when rescaling the image to make choice of compactness insensitive to the image values.
+  The new behavior makes it possible to mask values such as `numpy.nan` or `numpy.infinity`.
+  Additionally, raise an error if the input ``image`` has two dimensions and a ``channel_axis`` is specified - indicating that the image is multi-channel
+  (`#6525 <https://github.com/scikit-image/scikit-image/pull/6525>`_).
+- Fix unexpected error when passing a tuple to the parameter ``exclude_border`` in ``skimage.feature.blog_dog`` and ``skimage.feature.blob_log``
+  (`#6533 <https://github.com/scikit-image/scikit-image/pull/6533>`_).
+- Raise a specific error message in ``skimage.segmentation.random_walker`` if no seeds are provided as positive values in the parameter ``labels``
+  (`#6562 <https://github.com/scikit-image/scikit-image/pull/6562>`_).
+- Raise a specific error message when accessing region properties from ``skimage.measure.regionprops`` when the required  ``intensity_image`` is unavailable
+  (`#6584 <https://github.com/scikit-image/scikit-image/pull/6584>`_).
+- Avoid errors in ``skimage.feature.ORB.detect_and_extract`` by breaking early if the octave image is too small
+  (`#6590 <https://github.com/scikit-image/scikit-image/pull/6590>`_).
+- Fix ``skimage.restoration.inpaint_biharmonic`` for images with Fortran-ordered memory layout (`#6263 <https://github.com/scikit-image/scikit-image/pull/6263>`_)
+
+Documentation
+-------------
+.. Everything regarding the documentation that is visible to users.
+
+New
+~~~
+- Add a textbook-like tutorial on measuring fluorescence at the nuclear envelope of a cell
+  (`#5262 <https://github.com/scikit-image/scikit-image/pull/5262>`_).
```

Comment:
```suggestion
```
This isn't new! It was added in v0.19.2 (see https://github.com/scikit-image/scikit-image/blob/main/doc/release/release_0.19.rst#scikit-image-0192).